web/packages/teleport/src/Login/Login.tsx

```diff
@@ -78,3 +90,114 @@ export function LoginComponent({
     </>
   );
 }
+
+const LicenseBox = styled(Box)`
+  width: 100%;
+  max-width: 550px;
+  padding: ${props => props.theme.space[4]}px;
+  background-color: ${props => props.theme.colors.levels.surface};
+  border-radius: ${props => props.theme.radii[3]}px;
+  margin: auto;
+`;
+
+function LicenseAcknowledgement({
+  setLicenseAcknowledged,
+}: {
+  setLicenseAcknowledged(): void;
+}) {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <>
+      <LogoHero />
+      <LicenseBox>
+        <Text
+          css={`
+            line-height: 32px;
+            font-weight: 500;
+            font-size: ${props => props.theme.fontSizes[7]}px;
+          `}
+        >
+          Welcome to Teleport
+        </Text>
+        <Text
+          css={`
+            font-weight: 300;
+            line-height: 24px;
+            font-size: ${props => props.theme.fontSizes[3]}px;
+            color: ${p => p.theme.colors.text.slightlyMuted};
+          `}
+        >
+          Companies may use Teleport Community Edition on the condition they
+          have less than 100 employees and less than $10MM in annual revenue. If
+          your company exceeds these limits, please{' '}
+          <Text
+            as="a"
+            href="https://goteleport.com/signup/enterprise/?utm_campaign=CTA_terms_and_conditions&utm_source=oss&utm_medium=in-product"
+          >
+            contact us
+          </Text>{' '}
+          to evaluate and use Teleport.
+        </Text>
+        <Flex mt={3} alignItems="center" gap={2}>
+          <StyledCheckbox
+            checked={checked}
+            onChange={e => {
+              setChecked(e.target.checked);
+            }}
+          />
+          By clicking continue, you agree to our{' '}
```

Comment:
Yeah this is weird. The font size/weight/lineheight are all the same as design but appearing slightly differently. I'll see whats going on with it.

Reply:
Talked to kenny. There are few things like letter spacing that are in the new typography design system that aren't implemented yet that sliiiightly change the styles of text. I'll leave this as is since its really close and the new typography updates should handle the rest